content/Arduino Cloud/Cloud Editor/If-you-see-an-error-when-performing-an-Over-the-Air-OTA-upload.md

```diff
@@ -0,0 +1,143 @@
+* An Over-the-Air upload will only be attempted if the ![OTA symbol](img/symbol_ota.png) OTA icon is visible to the right of the Upload button. If USB upload is also available, make sure that the OTA option is selected.
+
+In this article:
+
+* [Identifying the OTA error message](#identify-error)
+* [Error reference for OTA uploads](#errors)
+
+<!--
+
+TODO: Make list of URLs with anchors
+
+#sha256unknown
+#sha256mismatch
+#otastorageinitfail
+#serverconnecterrorfail
+#otaheadercrcfail
+#otadownloadfail
+#errorwriteupdatefilefail
+#httpresponsefail
+
+-->
+
+---
+
+## Identifying the OTA error message {#identify-error}
+
+When an Over-the-Air upload fails, you can find the error in the console output.
+
+Follow these steps:
+
+1. Find the console panel. If it is hidden, show it by clicking the button in the bottom-right corner.
+2. Find the line that starts with "SHA256" or "Board error".
+3. Find the instructions for that error in the [error reference](#errors).
+
+<!-- TODO: Update image -->
+
+<img src="img/ota-error-sha.png" width=800px/>
+
+---
+
+## Error reference for OTA uploads {#errors}
+
+In this section:
+
+* [Board error - errorwriteupdatefilefail](#errorwriteupdatefilefail)
+* [Board error - httpresponsefail](#httpresponsefail)
+* [Board error - otadownloadfail](#otadownloadfail)
+* [Board error - otaheadercrcfail](#otaheadercrcfail)
+* [Board error - otastorageinitfail](#otastorageinitfail)
+* [Board error - serverconnecterrorfail](#serverconnecterrorfail)
+* [SHA256 mismatch - Unable to apply the new firmware. Previous firmware detected after restart](#sha256mismatch)
+* [SHA256 Unknown - Unknown Firmware detected after restart.](#sha256unknown)
+
+### Board error - errorwriteupdatefilefail {#errorwriteupdatefilefail}
+
+This error indicates that the device's memory partition table is improperly formatted.
+
+Follow these steps:
+
+1. Run the memory formatting sketch for your device architecture.
+1. Reinitialize the storage with this Sketch: [QSPIFormat.ino](https://github.com/arduino/ArduinoCore-mbed/blob/main/libraries/STM32H747_System/examples/QSPIFormat/QSPIFormat.ino)
+1. Reinstall the SSL certificates with this Sketch: [WiFiFirmwareUpdater](https://github.com/arduino/ArduinoCore-mbed/tree/main/libraries/STM32H747_System/examples/WiFiFirmwareUpdater)
+
+> [!TIP]
+> If you're using GIGA R1 WiFi, see [Configure GIGA R1 WiFi for Over-The-Air (OTA) uploads](https://support.arduino.cc/hc/en-us/articles/12370721200540-Configure-GIGA-R1-WiFi-for-Over-The-Air-OTA-uploads).
+
+### Board error - httpresponsefail {#httpresponsefail}
+
+This error can occur if the OTA update fails during the firmware download.
+
+Follow these steps:
+
+1. Ensure that the board has a stable connection to Arduino Cloud.
+1. Avoid extensive calls to the `delay()` function or other blocking code inside the `loop()` function of your sketch.
+1. Try deactivating the watchdog timer on the device.
```

Comment:
I think we should add here a reference or a page where we explain how to deactivate the watchdog.

This is the line to use in the `setup()` function. The `false` does the trick.
`ArduinoCloud.begin(ArduinoIoTPreferredConnection,false);`

Reply:
We should also put this in all the sections of the document where the watchdog is mentioned.

Reply:
Resolved with https://github.com/arduino/help-center-content/pull/393/commits/d271aad09edba74f19019c4e01fd732f53ccea65